src/test/resources/test.template.properties

```diff
@@ -40,6 +40,12 @@ test.chromedriver.path=
 test.timeout=15
 
 ###############################################################################
+# Set to true to enable GodMode. Please read the GodMode documentation if you are
+# not yet familiar with it, and use with care. Remember to set back to false when
+# done and rerun the test(s).
+test.godmode.isenabled=false
```

Comment:
I think I will standardise to `GodMode` (one word), so `test.godmode.enabled` and `IS_GODMODE_ENABLED` (and in a few other places like the GodMode in CI error message). Thanks for pointing out!